core/src/main/java/org/elasticsearch/index/engine/VersionValue.java

```diff
@@ -32,8 +32,15 @@
     /** the version of the document. used for versioned indexed operations and as a BWC layer, where no seq# are set yet */
     private final long version;
 
-    VersionValue(long version) {
+    /** the seq number of the operation that last changed the associated uuid */
+    private final long seqNo;
```

Comment:
we can also just use the members no need to getters... I like it when it's struct like for internal things like this. WDYT?